aspnetcore/grpc/supported-platforms.md

```diff
@@ -101,8 +101,6 @@ The following table lists .NET implementations and their gRPC client support:
 | .NET Framework 4.6.1                         | ⚠️&dagger;        | ✔️         |
 | Blazor WebAssembly                           | ❌                | ✔️         |
 | Mono 5.4                                     | ❌                | ✔️         |
```

Comment:
@JamesNK, I believe we would remove Xamarin.IOS and Xamarin.Andriod here and not mention .NET MAUI in its place given this context correct?